package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "angular-books-crud-lab",
+  "version": "1.0.0",
+  "description": "<!-- Location: SF -->",
+  "main": "index.js",
+  "dependencies": {
+    "angular-route": "^1.6.2",
+    "bower": "^1.8.0",
+    "mongoose": "^4.8.4"
```

Comment:
I don't think you actually use these dependencies, and you should've been using budo rather than node. Why do you have a package.json file?